### DISCUSSION TITLE
如何在OB中把一篇已经存在的笔记粘贴为双链

### DISCUSSION THREAD

**Discussion by vipyanzg** [oldwinter/knowledge-garden#49] [提问]
感谢您的库，让我看懂了LYT和ACCESS，感谢。
我之前是思源笔记的用户，在思源中有个操作——我可以很方便的把一个已经存在的笔记复制为引用块，再粘贴到新笔记中成为一个[[双链]]，这个操作在OB中该如何实现呢？

**Comment by oldwinter** [comment]
额，你这个场景，ob似乎无法实现，可能有插件可以吧。我个人习惯都是在要引用的页面输入中括号直接搜索目标页面及其块，倒也觉得方便符合使用逻辑。
